services/contact_us.js

```diff
@@ -2,13 +2,14 @@
 import aws from 'aws-sdk'; // eslint-disable-line import/no-unresolved, import/no-extraneous-dependencies
 import { validateAndSendEmail } from './ses';
 
-const contactUsEmailTo = process.env.CONTACT_US_EMAIL_TO;
-
-if (!contactUsEmailTo) {
-  throw new Error('CONTACT_US_EMAIL_TO environment variable not set!');
-}
 
 export default function doContactUs(event, cb) {
+  const contactUsEmailTo = process.env.CONTACT_US_EMAIL_TO;
+
+  if (!contactUsEmailTo) {
+    throw new Error('CONTACT_US_EMAIL_TO environment variable not set!');
+  }
```

Comment:
^ we were having a (possibly separate issue) that when that Env variable wasn't set, it was showing up on the front end when a user tries to use the mailing list form. Moving this into this function would prevent that issue.